loadtest/control/actions.go

```diff
@@ -191,9 +192,11 @@ func JoinTeam(u user.User) UserActionResponse {
 
 // CreatePost creates a new post in a random channel by the given user.
 func CreatePost(u user.User) UserActionResponse {
-	team, err := u.Store().RandomTeamJoined()
+	team, err := u.Store().CurrentTeam()
```

Comment:
Curious to know what was the issue here. It's changing from posting to any random team that the user is part of, to posting only in the current team. But that shouldn't be a bug. We can do it, but is it a bug though ?

Reply:
Looking at it now we should probably use the current channel directly. In this case it was more of a logic issue. It doesn't make much sense for a user to post to a team/channel he/she's not currently viewing.
I can update this.

Reply:
On a second thought I think we might want to revert these changes (using the current channel/team in these actions) until I figure out how to properly do this in the `SimulController` case.  
Keeping track of this data might not be trivial and also might break the `SimpleController` as soon as we order actions differently.